texar/data/data/multi_aligned_data.py

```diff
@@ -488,7 +490,8 @@ def collate(self, examples) -> Batch:
         batch: Dict[str, Any] = {}
         for i, transposed_example in enumerate(transposed_examples):
             kth_batch = self._databases[i].collate(transposed_example)
-            batch.update(kth_batch._batch)
+            for key, _ in self._names[i].items():
+                batch.update({self._names[i][key]: kth_batch[key]})
```

Comment:
Since we're already iterating over `.items()`, why not just write:
```python
for key, name in self._names[i].items():
    batch.update({name: kth_batch[key]})
```